release.cmake

```diff
@@ -65,6 +65,14 @@ MACRO(RELEASE_SETUP)
       && ! test x$$VERSION = x
         || (echo "Please set a version for this release" && false)
       && cd ${PROJECT_SOURCE_DIR}
+      # Update version in package.xml if it exists
+      && if [ -f "package.xml" ]; then
+          (echo "Updating package.xml to $$VERSION"
+           && sed -i \"s!<version>[0-9].[0-9].[0-9]</version>!<version>$$VERSION</version>!g\" package.xml
```

Comment:
Don't you need to add some `+` to the regex to support version `10.234.15423542` ?

Reply:
Great catch, thank you. I just added a commit to fix it